packages/cli/tsconfig.json

```diff
@@ -3,5 +3,14 @@
     "compilerOptions": {
         "outDir": "dist"
     },
-    "include": ["src/**/*"]
+    "include": ["src/**/*"],
+    "references": [
+        { "path": "../td-tools" },
+        { "path": "../core" },
+        { "path": "../binding-http" },
+        { "path": "../binding-file" },
+        { "path": "../binding-websockets" },
+        { "path": "../binding-coap" },
+        { "path": "../binding-mqtt" }
+    ]
```

Comment:
May I ask why this is necessary. Not just for this but also for other tsconfig files

Reply:
Yes, this is necessary, This is how the typescript transpiler knows how to cascade to the hierarchy of dependent projects.
The top level tsconfig.json should contain reference ptah to all major projects.
Inside a binding-xxxx project you would only have the path to the dependant project (namely, I guess, td-tools and core only).

Reply:
buid is failing due to master failing.  having (ff) #519 merged to master would help here.